concepts/00300-productExtensionExplicitProjection/README.adoc

```diff
@@ -0,0 +1,45 @@
+:icons: font
+:source-highlighter: prettify
+:doctype: book
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+== Description:
+
+Continues with the complex field example persisted as JSON. However, in this case, an explicit projection type is declared.
+
+== Goals:
+
+- Demonstrate custom mapping to/from projection
+- Demonstrate response only projection field
+- Show supporting admin customization
+- Demonstrate mapping to synthetic fields
+- Builds On : _00200-productExtensionComplexFieldJson_
```

Comment:
Can these "builds on" items be xrefs like in the root readme? Not sure if `..` works, but maybe `/concepts` would
```suggestion
- Builds On : xref:../00200-productExtensionComplexFieldJson/README.adoc[00200-productExtensionComplexFieldJson]
```